test/spec/endpoints.test.js

```diff
@@ -1,30 +1,31 @@
-/* global describe it before after beforeEach afterEach */
+/* global describe it before after afterEach */
 
 var app = require('../../')
-var request = require('supertest')(app)
 var nock = require('nock')
 
-let res = {
-  ok: true,
-  has_more: false,
-  messages: []
+/**
+ * Mock the commands because all we are testing is that they get run.
+ */
+let commands = {
+  'on': function commandOn () {
+    return Promise.resolve('On ran.')
+  },
+  'off': function commandOff () {
+    return Promise.resolve('Off ran.')
+  },
+  'level': function commandLevel () {
+    return Promise.resolve('level ran')
+  }
 }
+var request = require('supertest')(app(commands))
```

Comment:
Because of dependency injection I can mock the commands object and *just* test the routes without worrying about `nock` and whatnot.